pkg/plugin/predicates/predicate_manager_test.go

```diff
@@ -115,17 +115,17 @@ func TestPreemptionPredicates(t *testing.T) {
 	node.AddPod(victims[3])
 
 	// all but 1 existing pod should need removing
-	index, ok := predicateManager.PreemptionPredicates(pod, node, victims, 1)
-	assert.Check(t, ok, "check failed")
+	index := predicateManager.PreemptionPredicates(pod, node, victims, 1)
+	assert.Check(t, index != -1, "check failed")
 	assert.Equal(t, index, 2, "wrong index")
 
 	// try again, but with too many resources requested
 	pod = newResourcePod(framework.Resource{MilliCPU: 1500, Memory: 15000000})
 	pod.Name = "largepod"
 	pod.UID = "largepod"
 
-	index, ok = predicateManager.PreemptionPredicates(pod, node, victims, 1)
-	assert.Check(t, !ok, "check should have failed")
+	index = predicateManager.PreemptionPredicates(pod, node, victims, 1)
+	assert.Check(t, index == -1, "check should have failed")
 	assert.Equal(t, index, -1, "wrong index")
```

Comment:
Ditto.

-> assert.Equal(t, index, -1, "should not find any victim index after preemption check")